src/EventListener/ExceptionListener.php

```diff
@@ -13,14 +13,25 @@ class ExceptionListener
   public function onConsoleError(ConsoleErrorEvent $event) {
     $exitCode = $event->getExitCode();
     $error = $event->getError();
+    $errorMessage = $error->getMessage();
+
     // Make OAuth server errors more human-friendly.
     if ($error instanceof IdentityProviderException && $error->getMessage() === 'invalid_client') {
-      $event->setError(new AcquiaCliException('Your Cloud API credentials are invalid. Run acli auth:login to reset them.',
-        [], $exitCode));
+      $newErrorMessage = 'Your Cloud API credentials are invalid. Run acli auth:login to reset them.';
     }
 
     if ($error instanceof ApiErrorException) {
-      $event->setError(new AcquiaCliException('Acquia Cloud Platform API returned an error: ' . $error->getMessage(), [], $exitCode));
+      switch ($errorMessage) {
+        case "There are no available Cloud IDEs for this application.\n":
+          $newErrorMessage = $errorMessage . 'Delete an existing IDE (acli ide:delete) or submit a support ticket to purchase additional IDEs (https://support.acquia.com)';
```

Comment:
The process is to talk to your AM when you have one, and to talk to Acquia Sales otherwise. Also, if we do want to offer to submit a Support ticket, then we should use https://insight.acquia.com/support/tickets/new?product=p:ride which will select the IDE component automatically.

Reply:
To clarify, it's also fine to file a Support request as we will loop in the AM when there's one or give a path forward without one.

Reply:
What do you want the message to be then? Not sure how many options we want to provide.

Reply:
What about this?

> There are no available Cloud IDEs for this application.
>
> Delete an existing IDE (acli ide:delete) or contact your Account Manager or Acquia Sales to purchase additional IDEs.
You may also submit a support ticket to ask for more information (https://insight.acquia.com/support/tickets/new?product=p:ride)